src/pages/onlineDrawPage/const.ts

```diff
@@ -26,12 +27,13 @@ export const checkUserInRoom = async (data: Props) => {
       socket.emit(JOIN_ACCESS_SOCKET, { roomId, userId })
       setAccess(true)
     }
+
     setIsLoading(false)
     return true
   } catch (e) {
     if (e instanceof AxiosError) {
       if (e.response?.status === 409 || e.response?.status === 404) {
```

Comment:
Error codes should also be inside enum
for example
{
NOT_FOUND = 404
}

Reply:
Added 

```
export enum NetworkCodes {
  NOT_FOUND = 404,
  UNAUTHORITHED = 401,
  FORBIDDEN = 403,
  CONFLICT = 409
}
```